xtask/codegen/css.ungram

```diff
@@ -641,6 +642,16 @@ AnyCssAtRule =
 	| CssValueAtRule
 	| CssPositionTryAtRule
 	| CssViewTransitionAtRule
+	// Tailwind CSS 4.0 at rules
+	| CssThemeAtRule
+	| CssSourceAtRule
+	| CssUtilityAtRule
+	| CssVariantAtRule
+	| CssCustomVariantAtRule
+	| CssApplyAtRule
+	| CssReferenceAtRule
+	| CssConfigAtRule
+	| CssPluginAtRule
```

Comment:
Why aren't they named `Tw*`? It makes sense if they can be used when tailwind isn't enabled